pkg/proc/bininfo.go

```diff
@@ -381,28 +381,30 @@ func (bi *BinaryInfo) PCToLine(pc uint64) (string, int, *Function) {
 
 // LineToPC converts a file:line into a memory address.
 func (bi *BinaryInfo) LineToPC(filename string, lineno int) (pc uint64, fn *Function, err error) {
+	fileFound := false
 	for _, cu := range bi.compileUnits {
 		if cu.lineInfo.Lookup[filename] != nil {
-			pc = cu.lineInfo.LineToPC(filename, lineno)
+			fileFound = true
+			pc := cu.lineInfo.LineToPC(filename, lineno)
 			if pc == 0 {
 				// Check to see if this file:line belongs to the call site
 				// of an inlined function.
 				for _, ifn := range cu.concreteInlinedFns {
 					if strings.Contains(ifn.CallFile, filename) && ifn.CallLine == int64(lineno) {
-						pc = ifn.LowPC
-						fn = ifn.Parent
-						return
+						return ifn.LowPC, ifn.Parent, nil
 					}
 				}
 			}
-			fn = bi.PCToFunc(pc)
-			if fn != nil {
-				return
+			if fn := bi.PCToFunc(pc); fn != nil {
+				return pc, fn, nil
 			}
 		}
 	}
-	err = fmt.Errorf("could not find %s:%d", filename, lineno)
-	return
+	if fileFound {
+		return 0, nil, fmt.Errorf("could not find %s:%d", filename, lineno)
+	} else {
+		return 0, nil, fmt.Errorf("could not find statement at %s:%d, please use a line with a statement", filename, lineno)
```

Comment:
Wouldn't `"could not find file %s"` be clearer?

Reply:
It feels like the two messages should be flipped, right? If the file is found, then it is likely the user specified a location that didn't correlate to any actual code, thus this error message makes more sense for that case.

Reply:
Yes, that was a mistake - I"m sorry about the hasty commit, editing in github editor. I properly checked out the code and fixed the logic and and validated that I got the correct.

Reply:
I don't think the commit got applied appropriately, I'm still seeing the same code.